test/end-to-end/offer.js

```diff
@@ -150,7 +151,7 @@ describe("Offer endpoint tests", () => {
             test("Should successfully create an Offer", async () => {
                 const offer = {
                     title: "Test Offer",
-                    publishDate: "2019-11-17T00:00:00.000Z",
+                    publishDate: new Date(Date.now() - (24 * 3600 * 1000)),
```

Comment:
That magic constant is being used twice here and in other files as well, can you create a global `const DAY_TO_MS` with that value?

Reply:
done